.github/workflows/e2e-nvidia-t4-x1.yml

```diff
@@ -26,28 +37,27 @@ permissions:
   contents: read
 
 jobs:
-  start-runner:
-    name: Start external EC2 runner
+  e2e-workflow-ready:
+    permissions:
+      checks: read
+    uses: ./.github/workflows/status-checks.yml
+    with:
+      job_ids: >- # Space-separated job ids to wait on for status checks
+        DCO
+        actionlint
+        markdown-lint
+        shellcheck
+        spellcheck
+        lint-workflow-complete
+
+  start-ec2-runner:
```

Comment:
Particular reason to rename this?

Reply:
I needed to remove the `name:` element (since it has spaces) and thus changed the job id to be as descriptive as the remove name. It is not strictly necessary, but better to have more unique and descriptive job ids for use in the status-checks workflow.